chargeOclock/Source/Models/Client.swift

```diff
@@ -25,18 +25,24 @@ class Client {
 
 	init(name: String) {
 		self.name = name
+		self.isDirty = true
 	}
 
 	init(from json: [String: Any]) {
-		self.id = json["id"] as? Int ?? 0
+		self.id = json["id"] as? Int
 		self.name = json["name"] as? String ?? ""
+		self.isDirty = false
 	}
 
 	func toJSON() -> Data? {
```

Comment:
wait... where did the Data come from? missed this one... if the method is called toJSON, return a standard object, `[String: Any]`, not Data